src/test/java/it/gov/pagopa/payhub/activities/service/paymentsreporting/PaymentsReportingMapperServiceTest.java

```diff
@@ -92,27 +93,51 @@ void testMap2PaymentsReportings() throws DatatypeConfigurationException {
 	}
 
 	@Test
-	void testMap2TransferSemanticKeyDto() {
+	void testMap2TransferSemanticKeyWithOutComeCodeDTO() {
 		// Given
 		PaymentsReporting paymentsReporting = PaymentsReporting.builder()
 			.organizationId(1L)
+			.paymentOutcomeCode("OK")
 			.iuv("iuv123")
 			.iur("iur123")
 			.transferIndex(1)
 			.build();
 
 		// When
-		TransferSemanticKeyDTO result = mapper.map2TransferSemanticKeyDto(paymentsReporting);
+		TransferSemanticKeyWithOutComeCodeDTO result = mapper.map2TransferSemanticKeyWithOutComeCodeDTO(paymentsReporting);
 
 		// Then
 		assertEquals(1L, result.getOrgId());
 		assertEquals("iuv123", result.getIuv());
 		assertEquals("iur123", result.getIur());
 		assertEquals(1, result.getTransferIndex());
+		assertEquals("OK", result.getOutcomeCode());
 
 		TestUtils.checkNotNullFields(result);
 	}
 
+	@Test
+	void testMap2TransferSemanticKey() {
+		// Given
+		PaymentsReporting paymentsReporting = PaymentsReporting.builder()
+			.organizationId(1L)
+			.iuv("iuv123")
+			.iur("iur123")
+			.transferIndex(1)
+			.build();
+
+		// When
+		TransferSemanticKeyDTO result = mapper.map2TransferSemanticKeyWithOutComeCodeDTO(paymentsReporting);
+
+		// Then
+		assertEquals(1L, result.getOrgId());
+		assertEquals("iuv123", result.getIuv());
+		assertEquals("iur123", result.getIur());
+		assertEquals(1, result.getTransferIndex());
+
+		TestUtils.checkNotNullFields(result, "outcomeCode");
```

Comment:
configure the test in order to have the outcomeCode filled, in order to verify that it's correctly mapped (it should not be null)